contracts/utils/ShortStrings.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.8;
+
+import "./StorageSlot.sol";
+
+/**
+ * @dev Short string operations.
+ */
+library ShortStrings {
+    type ShortString is bytes32;
```

Comment:
I'd prefer to define this outside of the library, so we can just write `ShortString` when we use the type. Otherwise we will have to use `ShortStrings.ShortString`.

So far we've defined types inside libraries, but I think we need to chage that in 5.0.

Reply:
what about the custom error ?

Reply:
I would say the custom error should stay inside the library.